README_android.txt

```diff
@@ -47,18 +41,34 @@ Make NDK standalone toolchain
 Next you need to setup a standalone NDK toolchain. Set an environment
 variable to point to the desired location of the Android toolchain:
 
-    export ANDROID_NDK_TOOLCHAIN_ROOT=$HOME/android-toolchain
+    export ANDROID_NDK_TOOLCHAIN_ROOT=$HOME/android-toolchain-arm
+    export ANDROID_NDK_TOOLCHAIN_ROOT=$HOME/android-toolchain-arm64
+    export ANDROID_NDK_TOOLCHAIN_ROOT=$HOME/android-toolchain-x86
+    export ANDROID_NDK_TOOLCHAIN_ROOT=$HOME/android-toolchain-x86_64
+    export ANDROID_NDK_TOOLCHAIN_ROOT=$HOME/android-toolchain-mips
+    export ANDROID_NDK_TOOLCHAIN_ROOT=$HOME/android-toolchain-mips64
+
+If you do not want to distribute your game for all 6 supported Android
+architectures you can probably do it just for arm (most actual devices)
+amd x86_64 (the Android emulator).
 
-Assuming the NDK was extracted into $HOME/android-ndk run the following
+Assuming the NDK was extracted into ~/Android/Sdk/ndk-bundle run the following
 command:
 
-    $HOME/android-ndk/build/tools/make-standalone-toolchain.sh \
-        --platform=android-9 --install-dir=$ANDROID_NDK_TOOLCHAIN_ROOT
+    python ~/Android/Sdk/ndk-bundle/build/tools/make_standalone_toolchain.py \
+        --api=15 -install-dir=$ANDROID_NDK_TOOLCHAIN_ROOT --arch=arm
```

Comment:
--arch=arm doesn't work, as 'arm' is invalid according to the toolchain file. I had to do armabi. Should we change the readme or the toolchain file?

Reply:
"arm" is right, see here: https://developer.android.com/ndk/guides/standalone_toolchain.html

Note that there are two actual architectures using the same "arm" toolchain: armeabi and armeabi-v7a.

What is the error you got?

Reply:
I just tried here again and fixed a typo, -install-dir needs to be --install-dir. But "--arch=arm" seems to work for me.

Reply:
Whoops, my bad. I meant in the cmake command some lines below. It says `-DARM_TARGETS=arm`, but that doesn't work.